tool/checkstyle/rules.bzl

```diff
@@ -20,32 +20,40 @@ def _checkstyle_test_impl(ctx):
     opts = ctx.attr.opts
     sopts = ctx.attr.string_opts
 
-    args = ""
-    inputs = []
+    license_file = "tool/checkstyle/config/checkstyle-file-%s.txt" % ctx.attr.license_type
 
-    license_file = "external/vaticle_dependencies/tool/checkstyle/config/checkstyle-file-%s.txt" % ctx.attr.license_type
+    if ctx.workspace_name != "vaticle_dependencies":
```

Comment:
We have to support checkstyles existing in vaticle_dependencies, in which case the license file won't be found in `external/vaticle_dependencies` (because this is vaticle_dependencies! The checkstyle was coming from inside the house!)

Reply:
This hardcoded string check isn't ideal and seems a little fragile, but I haven't found a way to ask bazel if the rule implementation we're in is defined in the same workspace we're trying to check.